src/main/scala/GithubRelease.scala

```diff
@@ -11,6 +12,9 @@ case object GithubRelease {
   type DefTask[X] = Def.Initialize[Task[X]]
   type DefSetting[X] = Def.Initialize[Setting[X]]
 
+  val ghreleaseDefaultTokenEnvVar: String = "GITHUB_TOKEN"
+  val ghreleaseDefaultTokenFile: File = file(s"${getProperty("user.home")}/.github")
```

Comment:
I suggest to do one of the two things:
* move these values to the `keys` object (which is auto-imported)
* or keep them here, but then remove the `ghrelease` prefix

In the first case, users will be able to refer to them without any imports, and in the second case users will have to import them, but use a shorter name.

Right now you have to write `ohnosequences.sbt.GithubRelease.ghreleaseDefaultTokenFile` which is unnecessarily long.